src/platforms/javascript/guides/cordova/ionic.mdx

```diff
@@ -1,11 +1,19 @@
 ---
 title: Ionic
 sidebar_order: 201
+noindex: true
+notoc: true
```

Comment:
```suggestion
notoc: true
draft: false
```

Reply:
Oh I didn't notice you changed that flag, I'll rollback to only alter the draft: false.